source/brussels/tier2_hardware/hydra_hardware.rst

```diff
@@ -20,18 +20,19 @@ ivybridge_mpi    16      2x 10-core INTEL E5-2680v2 (Ivybridge)      256 GB  900
 broadwell        27      2x 14-core INTEL E5-2680v4 (Broadwell)      256 GB    1 TB      10 Gbps
 broadwell_himem  1       4x 10-core INTEL E7-8891v4 (Broadwell)      1.5 TB    4 TB      10 Gbps
 skylake          22      2x 20-core INTEL Xeon Gold 6148 (Skylake)   192 GB    1 TB      10 Gbps
-skylake_mpi      31      2x 20-core INTEL Xeon Gold 6148 (Skylake)   192 GB    1 TB      EDR-IB
+skylake_mpi      33      2x 20-core INTEL Xeon Gold 6148 (Skylake)   192 GB    1 TB      EDR-IB
 ===============  ======  ==========================================  ======  ==========  =======
 
 GPU nodes
 ---------
 
-===============  ======  ===============================  ==========  =======================================  ==========  ==========  =======
-Slurm partition  nodes   GPUs per node                    GPU memory  processors per node                      CPU memory  local disk  network
-===============  ======  ===============================  ==========  =======================================  ==========  ==========  =======
-pascal_gpu       4       2x Nvidia Tesla P100 (Pascal)    16 GB       2x 12-core INTEL E5-2650v4 (Broadwell)   256 GB      2 TB        10 Gbps
-ampere_gpu       8       2x Nvidia A100 (Ampere)          40 GB       2x 16-core AMD EPYC 7282 (Zen2)          256 GB      2 TB        EDR-IB
-===============  ======  ===============================  ==========  =======================================  ==========  ==========  =======
+==========================  ======  ===============================  ==========  =======================================  ==========  ==========  =======
+Slurm partition             nodes   GPUs per node                    GPU memory  processors per node                      CPU memory  local disk  network
+==========================  ======  ===============================  ==========  =======================================  ==========  ==========  =======
+pascal_gpu                  4       2x Nvidia Tesla P100 (Pascal)    16 GB       2x 12-core INTEL E5-2650v4 (Broadwell)   256 GB      2 TB HDD    10 Gbps
+ampere_gpu                  6       2x Nvidia A100 (Ampere)          40 GB       2x 16-core AMD EPYC 7282 (Zen2)          256 GB      2 TB HDD    EDR-IB
+ampere_gpu (big_local_ssd)  4       2x Nvidia A100 (Ampere)          40 GB       2x 16-core AMD EPYC 7282 (Zen2)          256 GB      5.9 TB SSD  EDR-IB
```

Comment:
Better move the *big_local_ssd* bit in its own column called *Features* to avoid users using that as a partition name.

Reply:
we should split the table in 2 parts as it's too much info to fit nicely, but that will be for another day.